src/main/java/org/myrobotlab/opencv/OpenCVFilterDL4J.java

```diff
@@ -18,13 +18,15 @@
 import org.bytedeco.opencv.opencv_core.IplImage;
 import org.bytedeco.opencv.opencv_core.Rect;
 import org.bytedeco.opencv.opencv_imgproc.CvFont;
+import org.junit.Ignore;
 import org.myrobotlab.logging.LoggerFactory;
 import org.myrobotlab.service.Deeplearning4j;
 import org.myrobotlab.service.Runtime;
 import org.myrobotlab.service.Solr;
 import org.slf4j.Logger;
 
 // FIXME - should be OpenCVFilterDl4j
+@Ignore

```

Comment:
I think this was meant to go on the test, and not the filter itself

Reply:
agreed..   @Ignore shouldn't be used in this class.

Reply:
:facepalm:

Reply:
removed